src/components/cc-addon-admin/cc-addon-admin.js

```diff
@@ -97,7 +96,7 @@ export class CcAddonAdmin extends LitElement {
   render () {
 
     const isFormDisabled = (this.error !== false) || this.saving;
```

Comment:
nitpick: minor simplification

```suggestion
    const isFormDisabled = this.error || this.saving;
```